OpenSourcePolitics/decidim-app
# Configure deepl service

#### :tophat: Description
This PR aims to enable Deepl service on the decidim-app, to allow some users to migrate on this application. 

#### Testing
* Configure env variables
* Enable in the Configuration panel
* See that machine translation is enabled

#### Tasks
- [ ] Fix seed error with decidim-templates
